An error occurred while executing the following cell:
------------------
%matplotlib inline
import matplotlib.pyplot as plt
import numpy as np
import pymc3 as pm
import pandas as pd
import theano
import seaborn as sns

sns.set_style("whitegrid")
np.random.seed(123)

url = "https://github.com/twiecki/WhileMyMCMCGentlySamples/blob/master/content/downloads/notebooks/radon.csv?raw=true"
data = pd.read_csv(url)
# data = pd.read_csv('../data/radon.csv')
data["log_radon"] = data["log_radon"].astype(theano.config.floatX)
county_names = data.county.unique()
county_idx = data.county_code.values

n_counties = len(data.county.unique())
------------------

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
/tmp/ipykernel_3880/1135320850.py in <module>
      2 import matplotlib.pyplot as plt
      3 import numpy as np
----> 4 import pymc3 as pm
      5 import pandas as pd
      6 import theano

ModuleNotFoundError: No module named 'pymc3'
ModuleNotFoundError: No module named 'pymc3'
